nautobot/core/ui/object_detail.py

```diff
@@ -275,7 +275,14 @@ def get_extra_context(self, context: Context):
 
 
 class FormButton(Button):
-    def __init__(self, form_id: str, link_name: str, template_path="components/button/formbutton.html", **kwargs):
+    def __init__(
+        self,
+        form_id: str,
+        link_name: str,
+        template_path="components/button/formbutton.html",
+        link_includes_pk=True,
```

Comment:
Should this move "up" one level further and be an attribute of `Button` rather than `FormButton`? I'm not seeing why the need for this flag is specific to FormButton.